src/documentation/Documentation.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+
+/**
+ * The Documentation component returns a simple div element containing an h1 heading. 
+ *  Add your documentation content or components within the div as needed.
+*/
+const Documentation = () => {
+    return (
+        <div>
+            <h1>Documentation Page</h1>
```

Comment:
Swap opening tag to:

`<Typography variant="h2">`

and closing tag to:

`</Typography>`